packages/swing-store-simple/package.json

```diff
@@ -5,14 +5,14 @@
   "parsers": {
     "js": "mjs"
   },
+  "type": "module",
```

Comment:
Ooh, does this work? I'm suspicious that this PR changes the type for this one package and none of the others. @kriskowal does this interact well/badly with the other ESM/`-r ESM` changes you're planning?

Reply:
Does https://github.com/Agoric/agoric-sdk/issues/527#issuecomment-841747122 answer these questions?

Reply:
> Ooh, does this work?

Yep!

> I'm suspicious that this PR changes the type for this one package and none of the others. 

I did that to demonstrate that incremental, bottom-up support of NESM is possible without having to throw away `-r esm`.

> @kriskowal does this interact well/badly with the other ESM/`-r ESM` changes you're planning?

In talking with @kriskowal, this is a fine step forward.